external/model-preparation-algorithm/mpa_tasks/utils/data_utils.py

```diff
@@ -44,27 +44,31 @@ def is_training_subset(subset):
         return subset.name in ["TRAINING", "VALIDATION"]
 
     def load_image_from_cache(filename: str, to_float32=False):
-        with open(filename, "rb") as f:
-            fcntl.flock(f, fcntl.LOCK_SH)
-            try:
-                cached_img = cv2.imread(filename)
-                if to_float32:
-                    cached_img = cached_img.astype(np.float32)
-                return cached_img
-            except Exception as e:
-                logger.warning(f"Skip loading cached {filename} \nError msg: {e}")
-            finally:
-                fcntl.flock(f, fcntl.LOCK_UN)
+        f = open(filename, "rb")
+        fcntl.flock(f, fcntl.LOCK_SH)
```

Comment:
Can we catch exceptions from these lines?

Reply:
Revised. Please review again. Thank you!